docs/x509/verification.rst

```diff
@@ -76,7 +76,8 @@ the root of trust:
     >>> with open(certifi.where(), "rb") as pems:
     ...    store = Store(load_pem_x509_certificates(pems.read()))
     >>> builder = PolicyBuilder().store(store)
-    >>> builder = builder.time(verification_time)
+    >>> # verification_time = datetime.now() # default
+    >>> # builder = builder.time(verification_time)
```

Comment:
You can't just comment this out, because then the doctest fails.

If you want to add a comment over the existing code, that's fine.

Reply:
lets just remove `builder = builder.time(verification_time)`

in 99% of all use cases, `verification_time` is the default `datetime.now()`